static/directives/repo-view/repo-panel-tags.html

```diff
@@ -121,19 +121,19 @@ <h3 class="tab-header"><span class="hidden-xs">Repository </span>Tags</h3>
         style="width: 90px;">
       <a ng-click="orderBy('size')" data-title="The compressed size of the tag's image" data-container="body" bs-tooltip>Size</a>
     </td>
-    <td class="hidden-sm hidden-xs hidden-md"
-        ng-class="tablePredicateClass('expiration_date', options.predicate, options.reverse)"
+    <td class="hidden-sm hidden-xs"
```

Comment:
nit: Since we're about to deprecate the old UI, do we want to leave it as is and not modify the table?